### PR TITLE
Add git crypt for secrets

### PR DESCRIPTION
**Pull request by ChrisLovering**

### CODE REVIEW

**Review by HassanAbouelela — APPROVED** [review]
This is the most amazing pull-request I've seen in my time at the DevOps team. Seeing as #157 is not resolved, I am unable to run the one command necessary to review half the files in this PR, and thus can only give a half review.

File: README.md [hunk 1]
```diff
@@ -2,24 +2,16 @@
...
 
-The easiest way to do this is to use `kubectl create secret`. For example:
+The [.gitattributes](.gitattributes) is used to determine which files to encrypt. See [git-crypt](https://www.agwa.name/projects/git-crypt/) documentation for more information.
 
-```shell script
-kubectl create secret generic new-secret-env \
-  --from-literal=MY_FAVORITE_THING=lemons \
-  --from-literal=SUPER_LAME_DOG=snoopy
-```
+To work with our secrets, you must have your GPG key added by a member of the devops team. Once that is done, you have use git-crypt as documented.
```
Comment (HassanAbouelela) [inline README.md:10]: Check the last sentence here.

Comment (ChrisLovering) [reply README.md]: s/have/can https://github.com/python-discord/kubernetes/compare/8143937f27b6b131dd278d51b797473d6e92a7d0..e6f3cd69d6852c9beb148f49bb99b23dc27c0427

File: .gitattributes [hunk 2]
```diff
@@ -1 +1,3 @@
 * text=auto
+secrets.yaml filter=git-crypt diff=git-crypt
```
Comment (jchristgit) [inline .gitattributes:2]: Should we add `.yml` as well just to be sure?

Comment (ChrisLovering) [reply .gitattributes:2]: Yea, that would be a good idea to be safe

Comment (ChrisLovering) [reply .gitattributes:2]: https://github.com/python-discord/kubernetes/pull/156/commits/8ec8aac9b25c67dc221b716d0c3750d0a5a30e20

**Comment by ChrisLovering** [comment]
@jchristgit

**Pull request merged**